hw/top_earlgrey/data/clocks.xdc

```diff
@@ -0,0 +1,23 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## Clock Signal
+create_clock -add -name sys_clk_pin -period 10.00 -waveform {0 5} [get_ports IO_CLK]
+
+## Clock Domain Crossings
+create_generated_clock -name clk_50_unbuf -source [get_pin clkgen/pll/CLKIN1] [get_pin clkgen/pll/CLKOUT0]
+create_generated_clock -name clk_48_unbuf -source [get_pin clkgen/pll/CLKIN1] [get_pin clkgen/pll/CLKOUT1]
```

Comment:
since this comes out of the FPGA PLL, it might be better to fetch the clocks that are automatically created by the tool in order to create the clock groups below:
`get_clocks -of_objects [get_pins clkgen/pll/CLKOUT0]`
`get_clocks -of_objects [get_pins clkgen/pll/CLKOUT1]`

Reply:
do you mean when setting the relationships for clock groups?

Reply:
either that, or you save the clock names into a variable first.

that way you can use the "real" names for these clocks that the tool autogenerates (instead of having to define redundant generated clocks).

Reply:
updated!